.github/workflows/run_functional_product_tests.yaml

```
@@ -0,0 +1,60 @@
+# Purpose: Run the product functional tests every night
+
+name: Nightly Product Functional Tests
+
+# Run this workflow at midnight 
+# on every Sun to Thr (b/c GMT -5)
+on:
+  schedule:
+  - cron: '0 4 * * 0-4'
+  workflow_dispatch: 
+
+jobs:
+  sharepoint-test:
+    name: SharePoint Test
+    runs-on: windows-latest
+    defaults:
+      run:
+        shell: powershell 
+    steps:
+    - name: Checkout Repo
+      uses: actions/checkout@v4
```

Steps takes an array of object - needs aditinal indentation 
```suggestion
      - name: Checkout Repo
        uses: actions/checkout@v4
```

---

Done.